server/pulp/server/content/web/views.py

```diff
@@ -144,4 +148,13 @@ def get(self, request):
             return self.redirect(request, self.key)
 
         # NotFound
-        return HttpResponseNotFound(request.path_info)
+        raise Http404
+
+    @staticmethod
+    def directory_index(request):
+        listing = os.listdir(request.path_info)
```

Comment:
@seandst suggested that we should compare os.normpath with os.path_info to see if someone is up to something. I am up to something. Right now.